templates/vue/src/components/TapestryMain/TapestryNode/index.vue

```diff
@@ -282,6 +284,12 @@ export default {
         .filter(n => n.status !== "draft")
       return rows.filter(row => row.completed).length / rows.length
     },
+    highlightNode() {
```

Comment:
should this be `this.node.reviewStatus == nodeStatus.ACCEPT &&
        this.settings.showAcceptedHighlight`